components/core/src/EncodedVariableInterpreter.hpp

```diff
@@ -62,13 +60,13 @@ class EncodedVariableInterpreter {
      * @param encoded_var
      * @return true if was successfully converted, false otherwise
      */
-    static bool convert_string_to_representable_double_var (const std::string& value, encoded_variable_t& encoded_var);
+    static bool convert_string_to_representable_float_var (const std::string& value, encoded_variable_t& encoded_var);
```

Comment:
Let's change the comment as well:
* double -> float
* For simplicity, I think we can remove the description of what "a representable double" is.